scripts/verify-install.sh

```diff
@@ -9,17 +9,23 @@ MANIFEST_SELECTOR=". | select(.name | test(\"manifest-v.*(\\\\.txt)$\")) | .name
 SIGNATURE_SELECTOR=". | select(.name | test(\"manifest-.*(\\\\.sig)$\")) | .name"
 HEADER_JSON="Accept: application/json"
 HEADER_GH_JSON="Accept: application/vnd.github.v3+json"
+MIN_REQUIRED_SIGNATURES=5
 
-# All keys that can sign lnd releases. The key must be downloadable/importable
-# from the URL given after the space.
+# All keys that can sign lnd releases. The key must be added as a file to the
+# keys directory, for example: scripts/keys/<username>.asc
+# The username in the key file must match the username used for signing a
+# manifest (manifest-<username>-v0.xx.yy-beta.sig), otherwise the signature
+# won't be counted.
```

Comment:
Not sure if useful to mention, maybe for educational value: Verifiers of the script MUST check that each fingerprint and user name is unique.